bundles/net.tourbook.device/src/net/tourbook/device/messages.properties

```diff
@@ -7,9 +7,9 @@ FitLog_Error_InvalidStartTime       = Tour start time is not set in {0}
 FitLog_Equipment_Name_Not_Available = N/A
 
 Port_Listener_Error_ntd001 = Serial Port Error
-Port_Listener_Error_ntd002 = The serial port could not be opened. \n\
+Port_Listener_Error_ntd002 = The serial port could not be open. \n\

```

Comment:
I think that this modification is changing the text wrongly. I've not checked the other `open(ed)` modifications it is just time consuming for me.

I was also not sure if  `R-R ... Beat ...` texts should/must be replaced with `Heart rate` why I didn't changed it in these cases.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/a952a084-d157-4440-a11b-c436907ab86c)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/72a542bd-8959-4b8d-b734-79e5980cba29)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/73c776ff-84c7-4bbe-b61b-aa79bb6b7ebf)